chia/cmds/cmds_util.py

```diff
@@ -10,3 +27,131 @@ def transaction_submitted_msg(tx: TransactionRecord) -> str:
 
 def transaction_status_msg(fingerprint: int, tx_id: bytes32) -> str:
     return f"Run 'chia wallet get_transaction -f {fingerprint} -tx 0x{tx_id}' to get status"
+
+
+async def validate_client_connection(
+    rpc_client: RpcClient, node_type: str, rpc_port: int, fingerprint: Optional[int]
+) -> Tuple[Optional[RpcClient], Optional[int]]:
```

Comment:
Not required, however it makes the code look a lot cleaner

Reply:
How is `x, y = f(x)` a lot cleaner than `y = f(x)`?

Reply:
I meant like we don't have to do it in the function, but if we want to use that function for that we have to

Reply:
the alternative is https://github.com/Chia-Network/chia-blockchain/pull/12532/commits/e844b8b9d94d36e5e25c78704d3b5e402671fef1#diff-1e43439572b349526b913914815647ba42e0d57b86edb4f0e18e1ea6d812033cR64

Reply:
the alternative is 
```
            if node_client_type == WalletRpcClient:
                wallet_and_f = await get_wallet(node_client, fingerprint=fingerprint)
                if wallet_and_f is None:
                    yield None, config, fingerprint
                node_client, fingerprint = wallet_and_f
            yield node_client, config, fingerprint
```

Reply:
If you still think that is better i'll do that

Reply:
I believe that right now it is called as:

```python
o_node_client, fingerprint = await validate_client_connection(node_client, node_type, rpc_port, fingerprint)
yield o_node_client, config, fingerprint
```
and it could instead be:
```python
fingerprint = await validate_client_connection(node_client, node_type, rpc_port, fingerprint)
yield node_client, config, fingerprint
```

Reply:
How do we validate if the node client works then?

Reply:
ah i found a way, thanks!